docs/source/process.mdx

```diff
@@ -244,6 +244,44 @@ Dataset({
 
 Notice how the subfields are now their own independent columns: `answers.text` and `answers.answer_start`.
 
+## Align
+
+The [`~datasets.Dataset.align_labels_with_mapping`] function is used to align a dataset label id with the label name, or you can also assign a different mapping of label to ids. 
+
+For example, the [Poem Sentiment](https://huggingface.co/datasets/poem_sentiment) dataset uses the following label mapping:
+
+```py
+>>> from datasets import load_dataset
+
+>>> poem = load_dataset("poem_sentiment")
+>>> labels = poem["train"].features["label"].names
+>>> label2id, id2label = dict(), dict()
+>>> for i, label in enumerate(labels):
+...     label2id[label] = str(i)
+...     id2label[str(i)] = label
```

Comment:
IMO this is cleaner:
```python
id2label = {i, label for i, label in enumerate(labels)}
label2id = {label, i for i, label in id2label}
```

Reply:
Looked at the original paper, and it seems like they just dropped all samples with the `mixed` label from the final version of the dataset. I made a note of that, and added that we won't do any additional preprocessing to also drop the `mixed` label!

Reply:
Thanks for the investigation!